tasks/managed/push-snapshot/push-snapshot.yaml

```
@@ -181,9 +185,39 @@ spec:
             # shellcheck disable=SC2128
             jq -s 'reduce .[] as $item ({}; . * $item)' \
               "$SOURCE_AUTH_FILE" "$DEST_AUTH_FILE" > "$DOCKER_CONFIG"/config.json
+            
+            # Check if we should copy attached artifacts
+            copy_artifacts="$COPY_ATTACHED_ARTIFACTS"
+            if [[ "$copy_artifacts" == "true" ]]; then
+              # Check for any attached artifacts using oras discover
+              printf '* Checking for attached artifacts on %s\n' "$3"
+              if oras discover \
+                --registry-config "$SOURCE_AUTH_FILE" \
+                "$3" \
+                --format json \
+                >/tmp/artifacts.json 2>/dev/null; then
+                artifact_count=$(jq -r '.manifests | length' /tmp/artifacts.json 2>/dev/null || echo "0")
```

Again, why hide error output?

---

Fixed, thanks.